apps/docs/quickstart/apis/hono.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: Hono
+description: API Authentication in Hono
+mode: "wide"
+---
+
+## Prerequisites
+
+- Created your [Unkey account](https://app.unkey.com/auth/sign-up)
+- Created an API in the [Unkey dashboard](https://app.unkey.com/apis)
+
+<Card title="Skip the tutorial" icon="link" href="https://github.com/unkeyed/examples/tree/main/hono">
+  Don't want to read the tutorial? Click this to get an example ready to test.
+</Card>
+
+<Steps titleSize="h3">
+
+<Step title="Create Hono Application">
+Run the following command to create your Hono project
+
+<Tabs>
+  <Tab title="npm">
+  ```bash
+  npm create hono@latest
+  ```
+  </Tab>
+  <Tab title="pnpm">
+  ```bash
+  pnpm create hono@latest
+    ```
+  </Tab>
+  <Tab title="yarn">
+  ```bash
+  yarn create hono@latest
+    ```
+  </Tab>
+  <Tab title="bun">
+  ```bash
+    bun create hono@latest
+  ```
+  </Tab>
+</Tabs>
+</Step>
+<Step  title="Install">
+
+Now install the `@unkey/hono` package
+
+<Tabs>
+  <Tab title="npm">
+  ```bash
+    npm install @unkey/hono
+    ```
+  </Tab>
+  <Tab title="pnpm">
+  ```bash
+    pnpm add @unkey/hono
+    ```
+  </Tab>
+  <Tab title="yarn">
+  ```bash
+    yarn add @unkey/hono
+    ```
+  </Tab>
+  <Tab title="bun">
+  ```bash
+    bun install @unkey/hono
+    ```
+  </Tab>
+</Tabs>
+</Step>
+
+<Step title="Update index to use our Hono SDK">
+
+Create a new route and add the following code
+
+```ts /src/index.ts
+import { Hono } from "hono";
+import { unkey, UnkeyContext } from "@unkey/hono";
+
+const app = new Hono<{ Variables: { unkey: UnkeyContext } }>();
+
+app.use("*", unkey());
+
+app.get("/", (c) => {
+	return c.text("Hello Hono!");
+});
+
+export default app;
+```
+
+</Step>
+
+<Step title="Running it">
+
+<Tabs>
+  <Tab title="bun">
+  ```bash
+    bun run dev
+    ```
+  </Tab>
+  <Tab title="pnpm">
+  ```bash
+    pnpm run dev
+    ```
+  </Tab>
+  <Tab title="yarn">
+  ```bash
+    yarn run dev
+    ```
+  </Tab>
+  <Tab title="npm">
+  ```bash
+    npm run dev
+    ```
+  </Tab>
+</Tabs>
+</Step>
+
+<Step  title="Try it out">
+
+Go to https://app.unkey.com and create a new key. Then verify it with our new server:
+
+```bash
+curl -XPOST 'http://localhost:8787/' \
+  -H "Authorization: Bearer <KEY>"
+```
+
+It should return `Hello Hono!"`
+
+</Step>
+
+</Steps>
+
+## What is next?
```

Comment:
fyi, mintlify has a concept of "snippets", where you could put this common footer and then reference it